wiki/Difficulties/osu!/en.md

```diff
@@ -0,0 +1,10 @@
+# osu!
+
+There are the five difficulties a [beatmapper](/wiki/beatmapper) can [map](/wiki/map) for an osu!standard [mapset](/wiki/mapset).
```

Comment:
but there are only 5 [categories]: Easy, Normal, Hard, Insane, and Expert; as of now